package.json

```diff
@@ -27,7 +27,7 @@
     "react-bootstrap": "0.32.4",
     "react-dom": "^16.8.6",
     "react-redux": "^7.1.0",
-    "react-router": "^3.2.0",
+    "react-router-dom": "^5.0.1",
```

Comment:
`react-router`? Nope. Now it's `react-router-dom` which has `react-router` as a dependency now.